### PR TITLE
Commerce emails3 - resubmit

### PR DESCRIPTION
**Pull request by justinchoi001**
This site has been rebuilt

### CODE REVIEW

File: docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/product-attachments.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Product Attachments
```
Comment (jrhoun) [inline docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/product-attachments.md:1]: Hey @justinchoi001 should this article be apart of this pull?

Comment (justinchoi001) [reply docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/product-attachments.md]: I have removed the article and screenshots.

**Comment by justinchoi001** [comment]
Hi JR, I have removed the extra files.

**Comment by jrhoun** [comment]
Just pushed this up. Please review https://github.com/liferay/liferay-learn/commit/9114a1aa0857f76e73d68a43befd9213afcb8fbd for my changes. Take particular care to maintain consistent formatting in the landing htmls.

**Pull request closed without merging**